.gitlab-ci.yml

```diff
@@ -51,8 +51,8 @@ stages:
   - r_release_resources
   - l_build_and_test
   - b_build_and_test
-  - c_build_and_test
-  - multi_project
+# - c_build_and_test
+# - multi_project
```

Comment:
multi_project should remain active to trigger rajaperf builds

Reply:
Ooops. Sorry about that. Changed pushed to turn that back on.